### PR TITLE
Try to fix Ilya's branch

### PR DESCRIPTION
**Pull request by Gaurav0**

### CODE REVIEW

File: tests/helpers/wait-for-loaded-iframe.js [hunk 1]
```diff
@@ -33,24 +36,45 @@ export default async function(app, url) {
...
-  }
-  await tryVisit();
+    run(waitForContents);
+  }));
 
   await new RSVP.Promise(function (resolve) {
-    run.later(resolve, 10);
+    run.later(resolve, 1000);
   });
 }
```
Comment (knownasilya) [inline tests/helpers/wait-for-loaded-iframe.js]: This was the helper I was unsure about. Thanks!

File: tests/acceptance/code-comment-test.js [hunk 2]
```diff
@@ -48,7 +49,7 @@ module('Acceptance | code comment', function(hooks) {
       metaKey: true
     });
 
-    [firstLine] = find('.CodeMirror-line');
+    firstLine = find('.CodeMirror-line');
     content = firstLine.textContent;
```
Comment (knownasilya) [inline tests/acceptance/code-comment-test.js]: Good catch in this commit!

File: tests/helpers/wait-for-loaded-iframe.js (hunk 1 above)
Comment (Gaurav0) [reply tests/helpers/wait-for-loaded-iframe.js]: Actually should revert that change.

**Comment by knownasilya** [comment]
🎉

**Pull request merged**